src/app/view/components/_common/dot-page-selector/service/dot-page-selector.service.ts

```diff
@@ -156,18 +150,21 @@ export class DotPageSelectorService {
     }
 
     private getPagesSearchQuery(param: string): { [key: string]: {} } {
-        debugger;
         const parsedUrl: DotSimpleURL = this.parseUrl(param);
 
-        let query = `${PAGE_BASE_TYPE_QUERY} +path:*${parsedUrl ? parsedUrl.pathname : param}*`;
+        let query = `${PAGE_BASE_TYPE_QUERY} +path:*${this.cleanPath(parsedUrl ? parsedUrl.pathname : param)}*`;
 
         if (this.currentHost) {
-            query += ` +conhostName:*${this.currentHost.hostname}*`;
+            query += ` +conhostName:${this.currentHost.hostname}`;
```

Comment:
Does this work when I'm looking just for a host, like if I have hosts: freddy1, freddy2, freddy3 and I search for "fred" does this return the 3 hosts?

Reply:
yes because host search is made in the method `getSites `, this is just specific for one host since we are checking `if (this.currentHost)`